src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -156,6 +156,7 @@ export default {
 	},
 	methods: {
 		submit() {
+			this.$kvTrackEvent('basket', 'Braintree Checkout', 'Button Click');
```

Comment:
To confirm - we're deviating from the updated event tagging scheme to be consistent with the related existing tagging. This isn't unreasonable, but from an analytics standpoint the gains from consistency are countered by making new non-conforming events. I would propose `this.$kvTrackEvent('basket', 'click', 'braintree-checkout-button');`, but I don't feel strongly about it.

Reply:
I'm happy to update this one seeing as it's new!